CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.2
+
+### Fixes
+
+- Updated the [Java Client Core Repository](https://github.com/Laserfiche/lf-api-client-core-java) dependency in pom.xml file to use the maven central version of rather than the `SNAPSHOT` version
```

Comment:
Could simplify this to something like
Updated the [Java Client Core Repository](https://github.com/Laserfiche/lf-api-client-core-java) dependency to release version 2.2.0

Reply:
ok done